requirements.txt

```diff
@@ -12,3 +13,6 @@ git+https://github.com/inducer/meshpy.git
 git+https://github.com/inducer/modepy.git
 git+https://github.com/inducer/meshmode.git
 git+https://github.com/inducer/grudge.git
+
+# Toy Fortran parser for Loopy
+f2py
```

Comment:
I don't think this exists. And even if, we want extra changes that I've made on my fork.

Reply:
Yeah, I changed it back already.